### DISCUSSION TITLE
Welcome to heizoel24-mex Discussions!

### DISCUSSION THREAD

**Discussion by darth-hp** [Secret-Lab-Productions/heizoel24-mex#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Olly77Wolly** [comment]
Hello,

I'm from Germany and have a question.
I myself have the meX from Heizoel24 and had just put the file in my HomeAssistant in the defined directory and restarted the HA. unfortunately I can't find a new integration. My HA runs on an Intel PC as a core variant.

**Reply by MartinStoffel** [reply]
Maybe you have time to check out my integration https://github.com/MartinStoffel/heizoel24mex

**Reply by Olly77Wolly** [reply]
Thank you very much. Works great.

**Comment by MartinStoffel** [comment]
Maybe you have time to check out my integration https://github.com/MartinStoffel/heizoel24mex
